boskos/janitor/compute.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"golang.org/x/net/context"
+	"google.golang.org/api/compute/v1"
+)
+
+
+func listInstances(computeService *compute.Service, project string, zone string) []*compute.Instance {
```

Comment:
it might also be worth defining an interface (named... Janitor? Cleaner? not sure) that we can implement for each resource type.

The AWS janitor sorta followed this idea:
https://github.com/kubernetes/test-infra/blob/ab64334a98d7e048a9b9b75cf220f01374dd3c4a/boskos/aws-janitor/resources/list.go#L23-L32

but I'm not sure whether that's exactly the right interface to use here.

Reply:
Added cleaners.go which is currently an interface defining only Create and Delete.
Hmm. Should add List. Will do that and push.

Reply:
Added List func as well to the interface.